catalogue_pipeline/transformer/src/main/scala/uk/ac/wellcome/transformer/transformers/MiroTransformableTransformer.scala

```diff
@@ -51,7 +51,10 @@ class MiroTransformableTransformer
           createdDate =
             getCreatedDate(miroData, miroTransformable.MiroCollection),
           subjects = getSubjects(miroData),
-          creators = getCreators(miroData),
```

Comment:
I think we need to keep doing this as creators are in the v1 api and we need to able to keep reindexing v1 stuff? We should probably add a contributors field in our internal work model and add the logic for populating that, but not remove the logic for adding creators

Reply:
Yep, so my plan is to do something like:

```scala
contributors = getCreators(miroData).map { Contributor(agent = _) }
```

or some level of wrapping to that effect.

But that has a knock-on effect on tests, and the diff gets substantially larger. For now I’ve taken it out so this code will compile, and I’m writing a separate patch in another branch which restores the transformations.

Reply:
Ah I see... Ok fair enough